src/adr/0002-make-taskmanager-distributed.md

```diff
@@ -0,0 +1,23 @@
+# 2. Make TaskManager Distributed
+
+Date: 2019-10-02
+
+## Status
+
+Accepted (lazy consensus)
+
+## Context
+
+In order to have a distributed version of James we need to have an unique way to deal with `Task`.
+
+Currently `TaskManager` is independent and node-exclusive.
```

Comment:
Currently, every james node of a cluster has its own instance of TaskManager and they have no knowledge of others, making it impossible to orchestrate task execution at the cluster level.
Tasks are scheduled and ran on the same node.